newsfragments/589.feature

```diff
@@ -0,0 +1 @@
+Add ``FormatNXmxEDeBIC``, which includes a mask specific for the SINGLA at eBIC.
```

Comment:
+= some "temporary change to reflect transient nature of detector present at Diamond / eBIC winter 2022"

So if / when we have one again in future we know these changes may not be wanted.